wire-ios-sync-engine/Source/UserSession/NSManagedObject+EncryptionContext.swift

```diff
@@ -21,10 +21,12 @@ import WireDataModel
 
 extension NSManagedObjectContext {
 
-    // TODO: [John] use flag here
-
     @objc
     public func deleteAndCreateNewEncryptionContext() {
-        zm_cryptKeyStore.deleteAndCreateNewBox()
+        proteusProvider.perform(
+            withProteusService: { _ in },
+            withKeyStore: { keyStore in keyStore.deleteAndCreateNewBox() }
+        )
```

Comment:
@typfel there's no core crypto api to clear the proteus data, should there be?

Reply:
There's no such API today, so you should would need delete underlaying sqlite store.

Reply:
Would it be useful to have such an API?

Reply:
I guess it's just to do some clean up, wouldn't it make sense from a security perspective to keep doing this?